src/test-integration/java/com/bettercloud/vault/util/DbContainer.java

```diff
@@ -1,35 +1,34 @@
 package com.bettercloud.vault.util;
 
-import org.junit.rules.TestRule;
-import org.junit.runner.Description;
-import org.junit.runners.model.Statement;
-import org.testcontainers.containers.Container;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.testcontainers.containers.GenericContainer;
-import org.testcontainers.containers.wait.HostPortWaitStrategy;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.wait.strategy.HostPortWaitStrategy;
+import org.testcontainers.lifecycle.TestDescription;
+import org.testcontainers.lifecycle.TestLifecycleAware;
 
-import java.io.IOException;
+import static org.junit.Assume.assumeTrue;
 
-public class DbContainer implements TestRule, TestConstants {
+public class DbContainer extends GenericContainer<DbContainer> implements TestConstants, TestLifecycleAware {
 
-    private final GenericContainer container;
+    private static final Logger LOGGER = LoggerFactory.getLogger(DbContainer.class);
+
+    public static final String hostname = "postgres";
 
     public DbContainer() {
-        container = new GenericContainer("postgres:11.3-alpine")
+        super("postgres:11.3-alpine");
+        this.withNetwork(CONTAINER_NETWORK)
+                .withNetworkAliases(hostname)
```

Comment:
This should fix up the database networking problem the two merges created. @denglertai @steve-perkins